.github/workflows/release.yaml

```diff
@@ -14,10 +14,17 @@ on:  # yamllint disable-line rule:truthy
           - major
           - minor
           - patch
+  repository_dispatch:
+    types: [release-go-libs]
 jobs:
+  package-update:
```

Comment:
can you add a comment above this step as to why its needed? Since this is slightly different from other release workflows for golibs, it can get confusing.

Reply:
Done.